build/docker/builder/cpu/ubuntu20.04/Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN mkdir -p /usr/local/go && wget -qO- "https://go.dev/dl/go1.20.7.linux-$TARGE
 
 # refer: https://code.visualstudio.com/docs/remote/containers-advanced#_avoiding-extension-reinstalls-on-container-rebuild
 RUN mkdir -p /home/milvus/.vscode-server/extensions \
-        /home/milvus/.vscode-server-insiders/extensions \
+    /home/milvus/.vscode-server-insiders/extensions \
```

Comment:
If there are no special requirements, try not to modify the Dockerfile as much as possible.

Reply:
ok. This is automatically formatted. I will disable this setting.